faucet/src/http.rs

```diff
@@ -211,35 +227,46 @@ async fn atomic_faucet_funding(
 
     ensure_funding_allowed(
         parachain_rpc,
-        account_id.clone(),
-        allowance_config,
+        &account_id,
+        &allowance_config,
         last_request_json,
         account_type.clone(),
     )
     .await?;
 
-    let mut transfers = vec![];
-    for AllowanceAmount { symbol, amount } in amounts.iter() {
-        let currency_id = CurrencyId::try_from_symbol(symbol.clone())?;
-        log::info!(
-            "AccountId: {}, Currency: {:?} Type: {:?}, Amount: {}",
-            account_id,
-            currency_id.symbol().unwrap_or_default(),
-            account_type,
-            amount
-        );
-        transfers.push(parachain_rpc.transfer_to(&account_id, *amount, currency_id));
+    if let Some(auth_url) = allowance_config.auth_url {
```

Comment:
seems like this should be inside `ensure_funding_allowed`?

Reply:
I don't think it matters too much but I can move it